.gitignore

```diff
@@ -0,0 +1,38 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore all environment files (except templates).
+/.env*
+!/.env*.erb
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/
+!/tmp/pids/.keep
+
+# Ignore storage (uploaded files in development and any SQLite databases).
+/storage/*
+!/storage/.keep
+/tmp/storage/*
+!/tmp/storage/
+!/tmp/storage/.keep
+
+/public/assets
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+# Ignore credentials
+c6CMp8hY2684ybgcOkT3QL/VG8fJtLCBvnyNdHSTGKldEnK2uRoCCkrbIWSG1kPbc97nY3qptcXSi3m00jtDvoq9bIjWaRAPWbhWQYWoUrjrFQcb9HfD9xYbViGoXWzre+82Qi1kQYleUAfHu7gn8V6Cx8q44FTDUCy9BexDIoa6boYNtNFFVPo5WSwrDOzxp2ERG3isGi9j+YgCqE0OvrWQlb9UqvW1sFW/r+EZoW2HEhDh69fSBwTGxCkGDYVvCdOaWsw1C5XPdA1hzZcg6LgzgFaD0vX1Iy1yjnqlqEMwj6iKPNyj/bmmEvPEAEPG4/9Lncgx64CmrzhCEGw3uDYtGEZdCo3bdQu/XAjte3MeiK2AtQXgiEGFzGtwHX9Odw2INqIJg+NN32mqJqnRqGyMft9V--YytIosuY99vbHs60--ZIIZTdVpuj40lblhI+h8ng==
```

Comment:
Thats not how you do it and also add a new line at the end of the file

Reply:
done